azerothcore/azerothcore-wotlk
# fix(DB): Lower Blackrock Spire creature import

## Changes Proposed:
- Imported NPCs from VMangos
- Removed Chromatic NPCs
- Added Creature Formations
- Patroling Smoldering Berseker

## Issues Addressed:

- Closes #9806
- Closes https://github.com/chromiecraft/chromiecraft/issues/2623

## SOURCE:

Wow Classic

## Tests Performed:


- Tested in-game


## How to Test the Changes:


1. .go c id 9568
2. Check for Creature formations
3. Check for Patroling NPC
4. No chromatic NPCs

## Known Issues and TODO List:


None


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
